subprojects/common/http-client/src/test/kotlin/com/avito/http/HttpClientProviderMetricsTest.kt

```diff
@@ -25,6 +25,7 @@ import java.net.SocketTimeoutException
 import java.util.concurrent.CountDownLatch
 import java.util.concurrent.TimeUnit
 
+@Flaky(reason = "MBS-11302")
```

Comment:
It doesn't work with a single `@Test`, see [AnnotationInspectorImpl](https://github.com/gradle/test-retry-gradle-plugin/blob/master/plugin/src/main/java/org/gradle/testretry/internal/filter/AnnotationInspectorImpl.java)

Reply:
Could we move the flaky scenario to a specific class then?

Reply:
I think it'd be better to fix the flakiness rather than splitting it into 2 classes. I can reproduce the failure locally, will try to fix it.

Reply:
Ok, moved for now. Will fix it later